playbooks/glance-cache-cleanup.yml

```diff
@@ -0,0 +1,52 @@
+---
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Stop glance services
+  hosts: glance_all
+  user: root
+  tasks:
+    - name: Stop glance-api
+      service:
+        name: glance-api
+        state: stopped
+        pattern: glance-api
+
+    - name: Stop glance-registry
+      service:
+        name: glance-registry
+        state: stopped
+        pattern: glance-registry
+
+    - name: Wipe glance cache directory
+      file:
+        state: absent
+        path: "/var/lib/glance/cache/"
+
+    - name: Generate cache directory
+      file:
+        state: directory
+        path: "/var/lib/glance/cache/"
+
+    - name: Restore glance-registry
+      service:
+        name: glance-registry
+        state: started
+        patter: glance-registry
```

Comment:
Looks like its running in gate now, but two cases where pattern is mispelled is stopping it from running.